core/client/fs/src/main/java/alluxio/client/file/AlluxioFileInStream.java

```diff
@@ -162,6 +167,19 @@ public int read() throws IOException {
           handleRetryableException(mBlockInStream, e);
           mBlockInStream = null;
         }
+        if (e instanceof OutOfRangeException) {
```

Comment:
Maybe put in a method to remove duplicate code?

Reply:
jiacheng's commit has put them in a method.